### PR TITLE
More testing for RefreshCaseStats

### PR DESCRIPTION
**Pull request by matthewblain**
Test that loading a snapshot of the casestats for 24dec2020 generates reasonable results.

## Screenshots

<details>
<summary>Screenshots</summary>
Add any relevant before/after screenshots here
</details>

## How did you test the change?

- [ ] iOS Simulator
- [ ] iOS Device
- [ ] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [x] other, please describe

Ran unit test.

## Checklist:

- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

File: server/appengine/src/main/java/who/RefreshCaseStatsServlet.java [hunk 1]
```diff
@@ -59,6 +59,10 @@ private static void updateData(
     if (timestamp > data.lastUpdated) {
       data.lastUpdated = timestamp;
     }
+    // Note that data.totalCases is a sum of dailyCases,
+    // while snapshot.totalCases is simply copied from the input data.
+    // The totalCases  at the most recent snapshot should match data.totalCases, but this
```
Comment (brunobowden) [inline server/appengine/src/main/java/who/RefreshCaseStatsServlet.java:64]: 1. double space after `totalCases`
1. newline before `data.totalCases` to make wrapping nicer on smaller screens

File: server/appengine/src/test/java/who/RefreshCaseStatsServletTest.java [hunk 2]
```diff
@@ -51,4 +49,29 @@ public void testParse() throws UnsupportedEncodingException {
...
+
+    RefreshCaseStatsServlet.JurisdictionData globalData = new RefreshCaseStatsServlet.JurisdictionData();
+    Map<String, RefreshCaseStatsServlet.JurisdictionData> countryData = new HashMap<>();
+
+    servlet.processWhoStats(rows, countryData, globalData);
+
+    assertEquals(1608768000000L, globalData.lastUpdated);
+    // The TotalCases is a good heuristic that the data loaded correctly: it is a sum
+    // of every daily case seen in the system so far.
+    assertEquals(77530799L, globalData.totalCases);
```
Comment (brunobowden) [inline server/appengine/src/test/java/who/RefreshCaseStatsServletTest.java:75]: This is a good one... but you add more. I was about to do add the same test:

Screen Shot 2020-12-25 at 9 08 38 PM

**Review by brunobowden — APPROVED** [review]

File: server/appengine/src/main/java/who/RefreshCaseStatsServlet.java (hunk 1 above)
Comment (matthewblain) [reply server/appengine/src/main/java/who/RefreshCaseStatsServlet.java:64]: haha, one of my editors has a bar at 80col and another at 100col. And i can't see the setting in android studio (though i see discussions of it online)

File: server/appengine/src/test/java/who/RefreshCaseStatsServletTest.java (hunk 2 above)
Comment (matthewblain) [reply server/appengine/src/test/java/who/RefreshCaseStatsServletTest.java:75]: if you send me some text i can put those in?

**Pull request merged**